stream/address/import.js

```diff
@@ -20,52 +19,30 @@ function streamFactory(db, done){
   // create a new stream
   return through.obj({ highWaterMark: 2 }, function( batch, _, next ){
 
-    // run serially so we can use transactions
-    db.serialize(function() {
+    // start transaction
+    db.transaction(() => {
 
-      // start transaction
-      db.run('BEGIN TRANSACTION', function(err){
+      // import batch
+      batch.forEach( function( address ){
 
-        // error checking
-        assert.transaction.start(err);
-
-        // import batch
-        batch.forEach( function( address ){
-
-          // insert points in address table
-          stmt.address.run(address, assert.statement.address);
-        });
+        // insert points in address table
+        stmt.address.run(address);
```

Comment:
if the statement were called insertAddressIntoTable then the code would be self documenting and a comment wouldn't be needed